### ISSUE TITLE
Getting the Panotopic Segmentation from Pretrained Model

### ISSUE THREAD

**Issue by StoicGilgamesh** [uber-research/UPSNet#108]
Thank you for this great repository! I am trying to reproduce the panoptic segmentation results using the pre-trained weights. I am using the following command:

python upsnet/upsnet_end2end_test.py --cfg upsnet/experiments/upsnet_resnet50_coco_16gpu.yaml --weight_path model/upsnet_resnet_50_coco_90000.pth

Which generates:  output['scores'], output['boxes'],output[ 'masks'], output['ssegs'], output['panos'],output[ 'cls_inds'],output[ 'pano_cls_inds']

But, when I checked with output['panos'], it does not generates segmentation results, but just normal semantic segmentation.  

Could you please help? How can panoptoic segmentation results can be obtained?
